docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_3.rst

```diff
@@ -76,6 +76,10 @@ Release notes: Open GEE 5.3.3
            - A memory management bug in XML processing was fixed,
              resulting in lower memory utilization and significantly improved
              performance.
+         * - 1577
+           - Use defusedxml in server code
+           - Added defusedxml as a dependecy.  Using defusedxml in place of
+             python xml.

```

Comment:
`python xml` -> `python XML`, I think. I might add `to improve security` at the end of the sentence as well.